lms/validation/_exceptions.py

```diff
@@ -0,0 +1,9 @@
+from pyramid import httpexceptions
+
+
+class ValidationError(
+    httpexceptions.HTTPUnprocessableEntity
+):  # pylint: disable=too-many-ancestors
+    def __init__(self, messages):
+        super().__init__()
+        self.messages = messages
```

Comment:
This is the custom exception class that `_handle_error()` above is wrapping webargs's exceptions in. In a separate `_exceptions.py` module to comply with our general code pattern of putting exceptions in per-package `exceptions.py` modules.

This is an `HTTPUnprocessableEntity` subclass so that validation errors will be HTTP 422 responses. 422 is the status code that webargs uses by default